charts/api/values.yaml

```diff
@@ -49,6 +49,10 @@ platform:
 
 wbstack:
   summaryInactiveThreshold:
+  # Ranges are passed in duration strings as expected by a PHP DateInterval
+  # constructor: https://www.php.net/manual/en/dateinterval.construct.php
+  # E.g. PT6H (6 hours) or P30D (30 days)
+  summaryCreationRateRanges: []
```

Comment:
Hmm if I put in some values here like in https://github.com/wmde/wbaas-deploy/pull/970/files#diff-7aa658a63aa196796483033ca1ab679d9130bbae12b5dc44b6ab8e1566e8fb3bR46

and run `helm template --debug .` I get an error:

```
install.go:178: [debug] Original chart version: ""
install.go:199: [debug] CHART PATH: /home/deer/wbaas/charts/charts/api


Error: template: api/templates/deployment-scheduler.yaml:63:3: executing "api/templates/deployment-scheduler.yaml" at <include "api.commonAppEnvVars" .>: error calling include: template: api/templates/_helpers.tpl:69:17: executing "api.commonAppEnvVars" at <','>: expected string; found ','
helm.go:88: [debug] template: api/templates/deployment-scheduler.yaml:63:3: executing "api/templates/deployment-scheduler.yaml" at <include "api.commonAppEnvVars" .>: error calling include: template: api/templates/_helpers.tpl:69:17: executing "api.commonAppEnvVars" at <','>: expected string; found ','
```

I get the same error when trying to apply this chart locally with https://github.com/wmde/wbaas-deploy/pull/970

Reply:
Good catch. Fixed it as suggested.